checker/tests/nullness/Issue880.java

```diff
@@ -0,0 +1,22 @@
+// Test case for Issue 880:
```

Comment:
Sounds good.  So then this test should either be renamed Issue989 or to the number of the new issue for storing annotations in extends/implements.